api/src/pcapi/utils/requests.py

```diff
@@ -27,25 +28,33 @@
 REQUEST_TIMEOUT_IN_SECOND = 10
 
 
+def _react_url(url: str) -> str:
```

Comment:
(typo) oups, c'est ça de jongler entre React et Python ! ;)

```suggestion
def _redact_url(url: str) -> str:
```